### DISCUSSION TITLE
Cannot listen to event htmx:responseError at window scope

### DISCUSSION THREAD

**Discussion by alarbada** [alpinejs/alpine#2916] [7. Bugs (archived)] [answered]
Hello there!

Maybe I'm doing something wrong, but as far as I know:

```html
	<div x-data="{ error: false }">
		<div x-on:htmx:responseError.camel.window="error = true"></div>
		<div x-show="error">
			Im a happy little error
		</div>
	</div>
```

This should work, but it doesn't with the current alpine cdn script.

At first I thought it could be something with the colon character in the htmx event, but I can listen to other events fired by htmx on the window element, so that gets discarted. And the camel modifier should work as expected.

I've checked that this works as expected:

```html
	<div x-data="{ error: false }">
		<div x-init="window.addEventListener('htmx:responseError', () => error = true)"></div>
		<div x-show="error">
			Im a happy little error
		</div>
	</div>
```

So I assume it must be something with how alpinejs interprets events.

Other people are also having this kind of error:

https://dev.to/preslavrachev/comment/1ijeh

Any ideas?

**Comment by SimoTod** [comment, marked as answer]
Attributes are case insensitive (which is way alpine has the camel modifier). You can't use `on:htmx:responseError`, I think it should be `on:htmx:response-error.camel`

**Reply by alarbada** [reply]
Finally tested it, and it works. But then the `.camel` modifier is quite useless if one must write `x-on:htmx:response-error.camel`, you better write `x-on:htmx:response-error` instead.

**Reply by SimoTod** [reply]
They are 2 different events. The event could be generated by a third party library (for example bootstrap) using a camel case name where you don't have control on the naming
